src/api/status/public/assets/css/material-dashboard.css

```diff
@@ -5006,6 +5006,8 @@ fieldset:disabled .btn {
   position: relative;
   display: flex;
   flex-direction: column;
+  height: 100%;
```

Comment:
I don't recommend overriding styles in the default asset, as it will affect the other components. You can add another stylesheet to override the default.

```css
/* homepage.css */
.card {
  height: 100%;
  justify-content: space-between
}
```

or make another class to make it clear (BEM style)
```css
/* homepage.css */
.card--homepage {
  height: 100%;
  justify-content: space-between
}
```

Reply:
Let's put `telescope-` in the name somehow, and maybe in a different directory, so it's clear that we are overriding the upstream styles.

Reply:
Let's put it in `/public/css/homepage.css`, and import it after the default assets. 

For the class name, `.card--dashboard` should be clear about the override. Breakdown: "card" is the component, "dashboard"  is the modifier. The modifier indicates that it must be used with the "card" class and goes after the "card" class (i.e. className="card card--dashboard"). This follows the popular BEM methodology, which makes it easy for new contributors to follow.